plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/controls/TextWithOpen.java

```diff
@@ -51,14 +51,16 @@ public class TextWithOpen extends Composite {
     private final Text text;
     private final ToolBar toolbar;
     private final boolean multiFS;
+    private final boolean binary;
 
     public TextWithOpen(Composite parent, boolean multiFS) {
-        this(parent, multiFS, false);
+        this(parent, multiFS, false, false);
     }
     
-    public TextWithOpen(Composite parent, boolean multiFS, boolean secured) {
+    public TextWithOpen(Composite parent, boolean multiFS, boolean secured, boolean binary) {
```

Comment:
To avoid breaking existing code that utilizes this API, I'd suggest making an overloaded version of the constructor.

Reply:
@KonstantinRight there are some comments that need to be addressed.

Reply:
Yeah, it seems reasonable. I've added an overloaded constructor just like you said.